sui/deploy-upgradable.js

```diff
@@ -0,0 +1,175 @@
+const { Command, Option } = require('commander');
+const { TxBuilder, updateMoveToml } = require('@axelar-network/axelar-cgp-sui');
+const { bcs } = require('@mysten/sui.js/bcs');
+const { fromB64, toB64 } = require('@mysten/bcs');
+const { saveConfig, printInfo, validateParameters, prompt, writeJSON } = require('../evm/utils');
+const { addBaseOptions } = require('./cli-utils');
+const { getWallet } = require('./sign-utils');
+const { loadSuiConfig } = require('./utils');
+
+async function upgradePackage(client, keypair, packageName, packageConfig, builder, options) {
```

Comment:
why is upgrade ability included in the same PR as adding a copy util? The goal of this PR is confusing

Reply:
there was a PR open that was used to deploy a package generically, so I tagged the copy util on it since I was gonna make the same function